src/App.tsx

```diff
@@ -35,7 +47,13 @@ function RNGalleryScreenWrapper({navigation}) {
   const Component = RNGalleryList[state.index].component;
   return (
     <View style={styles.container}>
-      <Button title="Menu" onPress={() => navigation.openDrawer()} />
+      <TouchableHighlight
+        style={styles.menu}
+        onPress={() => navigation.openDrawer()}
+        activeOpacity={0.5783}
```

Comment:
Why opacity? Should this be baked into the color? And should it relate to some defined `PlatformColor`?

Reply:
Sync'd offline